web/templates/compose_banner/upload_banner.hbs

```diff
@@ -1,5 +1,8 @@
 <div class="upload_banner file_{{file_id}} main-view-banner {{banner_type}}">
     <div class="moving_bar"></div>
     <p class="upload_msg">{{banner_text}}</p>
+    {{#if is_banner_info}}
+        <button class="upload_banner_cancel_button">{{t 'Cancel' }}</button>
+    {{/if}}
```

Comment:
Can we use a variable  that checks if this is a banner which tracks uploading a file? I can imagine `is_banner_info` being used for a different banner in the future.

Reply:
@amanagr , I have updated the code, but I feel like I didn't get your point clearly, could you please verify once if this is what you wanted or something else?

Reply:
Made a slight variable name change and how it should be procssed.